mydumper.c

```diff
@@ -2759,8 +2761,23 @@ guint64 dump_table_data(MYSQL * conn, FILE *file, char *database, char *table, c
 	GString* statement = g_string_sized_new(statement_size);
 	GString* statement_row = g_string_sized_new(0);
 	
+	query_tidb_rowid = g_strdup_printf("SELECT `_tidb_rowid` FROM `%s`.`%s` LIMIT 1", database, table);
```

Comment:
how about use a variable save "_tidb_rowid"

Reply:
no need

Reply:
Should I revert ?